README.md

```diff
@@ -1,7 +1,8 @@
 # Kottans-frontend
 ## Help with your first commit -> [discord](https://discord.gg/CqHKGU8vry)
 ---
-# Hello here!
+<!-- That's my first pull request -->
```

Comment:
Is it a markdown syntax?
Pls don't change other's people greetings.(it looks like you are trying update someone else "Hello")

Reply:
Okay, I understand. If I'll add my own greeting will the task be counted?